check-hazelcast-health.sh

```diff
@@ -2,8 +2,9 @@
 
 set -o errexit ${RUNNER_DEBUG:+-x}
 
-# shellcheck source=../hazelcast-docker/.github/scripts/abstract-simple-smoke-test.sh
-. hazelcast-docker/.github/scripts/abstract-simple-smoke-test.sh
+pushd hazelcast-docker
```

Comment:
Little confused. What folder is currently in when check-hazelcast-health.sh?

Reply:
It’s in the root of the hazelcast-packaging workflow. The docker one is checked out into a subdirectory, so we temporarily pop into there.